app/roster/Duncan_Grubbs/about.html

```diff
@@ -27,9 +27,10 @@
           <!-- Navigation -->
           <nav class="mdl-navigation">
             <a class="mdl-navigation__link" href="index.html">Home</a>
-            <a class="mdl-navigation__link" href="/#/about">About</a>
-            <a class="mdl-navigation__link" href="/#/contact">Contact</a>
-            <a class="mdl-navigation__link" href="/#/references">Resources</a>
+            <a class="mdl-navigation__link" href="about.html">About</a>
+            <a class="mdl-navigation__link" href="http://dhsdevelopers.org/#/join">Join</a>
```

Comment:
Why the absolute paths?  `/#/join` will work regardless of the host.  As it stands, your page only redirects to the production version of the site.